### DISCUSSION TITLE
Ask PDF questions

### DISCUSSION THREAD

**Discussion by heyivanvilla** [intellibar/main#8] [Ideas]
There are times where I want to ask chatGPT questions about a PDF. this could be a form, terms, whitepaper, etc.

Ideally I could have an upload file button or simply allow me to drop the PDF file into the text editor.

**Comment by astoilkov** [comment]
We are working towards that. In the next version, it will be possible to drag a text file and ask a question about it. Maybe the next step would be to support PDF format as well.

**Comment by rb81** [comment]
@astoilkov This could relate well with the requests to integrate IntelliBar with Ollama since you can download good embedding models that way. Assuming you want to build a RAG feature that is...

**Reply by astoilkov** [reply]
Yeah, we're are simultaneously experimenting with other models, Ollama, and RAG. These are all working experiments in dev. We should refine and at some point ship them.
